### ISSUE TITLE
Modify Search view to allow selection of multiple children, and default to all children for a logged-in user

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#1066]
# TL;DR

Create a smoother experience for families so that the first thing they see is the studies they are eligible for today.

# Narrative

As a new family (or as a family viewing the Search page), if I wasn't recruited for a specific study I have a slightly convoluted workflow to see the studies I could take: I have to select a kid from a drop down menu, and can only see one kid at a time. 

Instead, it would be great if the top bar had a 'tag' style selection, with all the families' children selected by default, e.g. 

Find studies for: [Fakelina]x [Fakelora]x [Bob]x

Where removing *all* children reverts to showing the full set of available studies, and the dropdown (or new UI element) shows checkboxes rather than requiring a single selection.
